### PR TITLE
4th task

### PR DESCRIPTION
**Pull request by cmcartaya**

### CODE REVIEW

File: cartaya/4/style4.css [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    display: inline-block;
+    color: #746767;
+    line-height: 20px;
+    margin-left: 5px;
+    text-decoration: none;
+    border-bottom: 1px solid;
+    margin-bottom: 6px;
+}
+
+fieldset  div  img {
```
Comment (reiniergs) [inline cartaya/4/style4.css]: element selector

**Review by reiniergs — CHANGES_REQUESTED** [review]

File: cartaya/4/task4.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="keywords" content="study,html5,css3,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <link rel="stylesheet" href="style4.css">
+    <title>Task#4</title>
+</head>
+<body>
+    <section class="section-left">
+        <h1 class="title">Directory Profile Name</h1>
+        <div class="profile-img"><img src="perfil.png" alt="profile picture" title="profile" height="152px" width="150.5"></div>
```
Comment (TahimiLeonBravo) [inline cartaya/4/task4.html]: That is not readable you need to put it in this way instead:
```
<div class="profile-img">
    <img src="perfil.png" alt="profile picture" title="profile" height="152px" width="150.5">
</div>
```

File: cartaya/4/task4.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="keywords" content="study,html5,css3,task">
+    <meta name="author" content="Carlos Miguel Cartaya Leon">
+    <link rel="stylesheet" href="style4.css">
+    <title>Task#4</title>
+</head>
+<body>
+    <section class="section-left">
```
Comment (TahimiLeonBravo) [inline cartaya/4/task4.html:13]: You need always use a convention to the names of all classes, it should create collisions.

File: cartaya/4/task4.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        <h1 class="title">Directory Profile Name</h1>
+        <div class="profile-img"><img src="perfil.png" alt="profile picture" title="profile" height="152px" width="150.5"></div>
+        <div class="profile-databox">
+            <h1 class="h1-profile-name">Profile Name</h1>
+            <div class="name profile-data">
+                <div class="profile-data-img"><img src="1.png" alt="name" width="25px" height="15px"></div>
```
Comment (TahimiLeonBravo) [inline cartaya/4/task4.html]: Don't use the width and height attribute intead use CSS styles. We need to follow the separation of concerns principle, the styles always in CSS.

File: cartaya/4/style4.css [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    left: -40px;
+    z-index: 1;
+}
+
+.section-left p, .section-left address  {
+    display: inline-block;
+}
+
+.title {
+    font-family: HelveticaNeueCE35Thin;
```
Comment (TahimiLeonBravo) [inline cartaya/4/style4.css]: You can define the font-family property  in the element container, this is a Inherited property you don’t need specify in each element inside container. If all the texts in the document have the same font, define this property only in the body or element container.

File: cartaya/4/style4.css [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+
+.profile-databox {
+    position: relative;
+    float: right;
+    display: inline-grid;
+    margin-right: 40px;
+    margin-top: 40px;
+}
+
+.h1-profile-name {
```
Comment (TahimiLeonBravo) [inline cartaya/4/style4.css]: Don't use the HTML elements name for classes name.

**Review by TahimiLeonBravo — APPROVED** [review]

**Pull request closed without merging**